### PR TITLE
Temporarily remove failing Bottlerocket upgrade test

### PR DESCRIPTION
**Pull request by cPu1**
### Description

Upgrading Bottlerocket nodegroups to 1.26 has been failing with this error:

```Couldn't proceed with upgrade process as new nodes are not joining node group bottlerocket-1```

This is due to https://github.com/bottlerocket-os/bottlerocket/issues/3013. 

This changelist temporarily removes the failing test until https://github.com/bottlerocket-os/bottlerocket/issues/3013 is out for managed nodegroups. 

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by stmcginnis** [comment]
Bottlerocket 1.13.3 was released yesterday, so these upgrades should now be working.

**Comment by cPu1** [comment]
> Bottlerocket 1.13.3 was released yesterday, so these upgrades should now be working.

Are these changes out for managed nodegroups as well? I understand that the SSM parameters have been updated but our last run of the integration tests, which happened after [this comment](https://github.com/bottlerocket-os/bottlerocket/issues/3013#issuecomment-1512193928) failed.

**Comment by stmcginnis** [comment]
There is some delay during the release due to SSM parameters being updated in waves to roll updates out. It should be getting the 1.13.3 version everywhere now. Any way to trigger a test and verify. If it fails, if there's any way to capture details about what was launched we could verify it's getting the expected version too.

File: integration/tests/managed/managed_nodegroup_test.go [hunk 1]
```diff
@@ -623,7 +623,6 @@ var _ = Describe("(Integration) Create Managed Nodegroups", func() {
 				}
 
 				upgradeNg(initialAl2Nodegroup)
-				upgradeNg(bottlerocketNodegroup)
```
Comment (Himangini) [inline integration/tests/managed/managed_nodegroup_test.go:626]:
```suggestion
// upgradeNg(bottlerocketNodegroup) See https://github.com/weaveworks/eksctl/issues/6544

**Comment by Himangini** [comment]
> There is some delay during the release due to SSM parameters being updated in waves to roll updates out. It should be getting the 1.13.3 version everywhere now. Any way to trigger a test and verify. If it fails, if there's any way to capture details about what was launched we could verify it's getting the expected version too.

@stmcginnis we have summed up the issue here https://github.com/weaveworks/eksctl/issues/6544 We're still experiencing issues with `1.13.3`

File: integration/tests/managed/managed_nodegroup_test.go (hunk 1 above)
Comment (cPu1) [reply integration/tests/managed/managed_nodegroup_test.go:626]: Done.

**Review by Himangini — APPROVED** [review]
👍🏻

**Pull request merged**